src/styles/blocks/button.scss

```
@@ -0,0 +1,23 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  height: 40px;
+  width: $width-card;
+  font-weight: $font-weight-bold;
+  background-color: #00acdc;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #fff;
+  border-radius: 5px;
+  border: none;
+
+  &:hover {
+    background-color: #fff;
+    color: #00acdc;
+    border: 1px solid#00ACDC;
```

```suggestion
    border: 1px solid #00ACDC;
```
Consider using variables for repeating colors